electron/windowTracker.js

```diff
@@ -60,6 +64,7 @@ const windowTracker = (windowName) => {
       width: windowState.width,
       height: windowState.height,
       isMaximized: windowState.isMaximized,
+      autoHideMenuBar: windowState.autoHideMenuBar ?? true,
```

Comment:
```suggestion
      autoHideMenuBar: windowState.autoHideMenuBar,
```

If this absolutely must be a real bool, you can add `!!` to the front.